.ergomake/docker-compose.yaml

```
@@ -0,0 +1,38 @@
+version: "3"
+services:
+  app:
+    build: ../
+    ports:
+      - "8080:8080"
+    environment:
+      BASE_URL: https://{{ services.api.url }}
```

You could put `http://localhost:8080` here just in case you want to run this locally with `docker compose up`
```suggestion
      BASE_URL: http://localhost:8080
```